axis.go

```diff
@@ -43,7 +43,8 @@ type Axis struct {
 		// Text is the axis label string.
 		Text string
 
-		// TextStyle is the style of the axis label text.
+		// TextStyle is the style of the axis label text. For the vertical axis,
```

Comment:
Can you shorten the line lengths here. Doc comments in structs and interfaces don't get flowed by godoc, so we need to be aware of that. Generally, shorter line lengths are easier to read in this context.

Also, I think changing the text to "For the vertical axis, one quarter turn counter clockwise will be added to the label text before drawing."